app/assets/stylesheets/farmbot.css.scss

```diff
@@ -80,7 +80,9 @@ $brown:          #CA8;
 .fb-tooltip{
   padding: 2px;
   display: inline;
+  font-style: normal;
   .tooltip-text {
+    font-style: normal;
```

Comment:
Properties should be ordered background-color, border-radius, color, display, font-style, padding